test/behaviour/typeql/TypeQLSteps.java

```diff
@@ -645,13 +644,13 @@ public boolean check(Concept concept) {
         }
     }
 
-    public static class AttributeUniquenessCheck {
+    public static abstract class AttributeUniquenessCheck {
 
         protected final Label type;
         protected final String value;
 
         AttributeUniquenessCheck(String typeAndValue) {
-            String[] s = typeAndValue.split(":");
+            String[] s = typeAndValue.split(":", 2);
```

Comment:
I'm not a huge fan of this but I think we'll take it for now. Thoughts: it's not obvious how this is going to interplace with the next assertion line, and what it means about the format of the 'typeAndValue' identifier we're expecting (less  explainable in 1 short sentence IMO)